src/Boundary/Boundary.cc

```diff
@@ -95,15 +95,14 @@ void
 Boundary<Dimension>::cullGhostNodes(const FieldList<Dimension, int>& flagSet,
                                     FieldList<Dimension, int>& old2newIndexMap,
                                     vector<int>& numNodesRemoved) {
-
-  auto& registrar = NodeListRegistrar<Dimension>::instance();
-  REQUIRE((int)numNodesRemoved.size() == registrar.numNodeLists());
+  const auto& nodeListPtrs = flagSet.nodeListPtrs();
+  REQUIRE(numNodesRemoved.size() == nodeListPtrs.size());
+  // auto& registrar = NodeListRegistrar<Dimension>::instance();
+  // REQUIRE((int)numNodesRemoved.size() == registrar.numNodeLists());
```

Comment:
I noticed when building in debug w/ the C++ libs as static that this would fail, the fix for registrar is addressed in an upcoming PR but I'm wondering if you saw this fail too?

Reply:
It built fine for me in debug mode. Maybe it is a case where the library name changes or something as is the case for qhull?